docs/install.rst

```diff
@@ -3,121 +3,120 @@
 Installation
 ============
 
-Here is a step by step plan on how to install Read the Docs.
+Here is a step by step guide on how to install Read the Docs.
```

Comment:
I'm not completely about this change, please let me know what you think.

Reply:
Better, for me.